app/views/widgets/lists.html

```diff
@@ -19,6 +19,20 @@
 
 {% endmacro %}
 
+{% macro taskStatus(content, title, status, previousTaskStatus) %}
+
+<div class="task-item-header">
+    <span class="heading-medium">{{title}}</span>
+    {% if status == 'complete' %}
+        <span class="task-completed">{{content.checkYourAnswers | upper}}</span>
```

Comment:
The css for the `phase-tag` mixin which is being included by `.task-complete` is styling this element as uppercase so we don't need to use the `upper` filter here.